backend/src/main/kotlin/clarity/backend/entity/ClassroomEntity.kt

```diff
@@ -104,13 +104,15 @@ class ClassroomEntity() {
         try {
             val statement = db!!.createStatement()
             val selectStatement = """
-                SELECT * FROM ClassroomStudents WHERE user_id = $userId
+                SELECT * FROM ClassroomStudents INNER JOIN Classroom 
```

Comment:
What does this updates query do

Reply:
it fetches the list of classes that the user does not own (i.e. the user is not the teacher of that class), only the classes that the students are part of.